app/io/pathfinder/websockets/WebSocketActor.scala

```diff
@@ -24,18 +30,48 @@ class WebSocketActor (
     controllers: Map[ModelType, WebSocketController]
 ) extends Actor {
     import WebSocketMessage._
+    Router
```

Comment:
This line should not go here. If we put it here, the Router object is not initialized until the first socket request is made. This causes the first socket connection to be delayed.